db/migrate/20180104234247_create_archived_backgrounds.rb

```diff
@@ -2,7 +2,7 @@ class CreateArchivedBackgrounds < ActiveRecord::Migration[5.1]
   def change
     create_table :archived_backgrounds do |t|
       t.bigint :archive_of
-      t.json :data
```

Comment:
I thought this was an old change.

Reply:
Yeah but there's no real danger in changing it. It will never be re-run in alpha or production. But it means that it can be run in the development env where the json type is not supported.